src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, Color color) {
+        this.radius = radius;
+        super.color = color;

```

Comment:
To set the color field inherited from the superclass 'Figure', you should use the constructor of the superclass. Instead of directly assigning a value to 'super.color', call 'super(color)' as the first statement in the Circle constructor.